cfgmgr/intfmgr.cpp

```diff
@@ -135,13 +141,29 @@ bool IntfMgr::doIntfGeneralTask(const vector<string>& keys,
             return false;
         }
 
-        setIntfVrf(alias, vrf_name);
-        m_appIntfTableProducer.set(alias, data);
+        // Set Interface VRF except for lo
+        if (!is_lo)
```

Comment:
Why are you excluding the loopbacks?

Reply:
Ip address for lo is configured in /etc/network/interfaces

Reply:
Not sure I follow your response. It is valid for a loopback to be in a vrf.

Reply:
@nikos-github , your comment is valid, but currently we do not support to have loopback in the vrf. We can add it in the future PR.